cnf-tests/mirror/images.json

```
@@ -1,7 +1,7 @@
 [
     {
         "registry": "quay.io/openshift-kni/",
-        "image": "cnf-tests:4.16"
+        "image": "cnf-tests:4.19"
```

```suggestion
        "image": "cnf-tests:4.18"
```

cnf-tests has not been branched yet:

```
$ docker pull quay.io/openshift-kni/cnf-tests:4.19
Error response from daemon: manifest for quay.io/openshift-kni/cnf-tests:4.19 not found: manifest unknown: manifest unknown
```

And openshift/release doesn't have any configuration for 4.19 yet.
We'll bump to 4.19 here when we'll branch out.

---

Done. Thanks for catching that